### DISCUSSION TITLE
使用ST_CONTAINS 函数查询时，为什么如下的sql两个结果不相同

### DISCUSSION THREAD

**Discussion by gongyongqiang123** [apache/doris#35207] [A - General / Q&A]
with tmp as (
SELECT  ST_ASTEXT(ST_Polygon("POLYGON ((116 30,117 30,117 32,116 32,116 30))")) ply , ST_ASTEXT(ST_Point(116.488065 ,30.662235))  po  
)
select    ST_CONTAINS(ST_GEOMETRYFROMTEXT(ply), ST_GEOMETRYFROMTEXT(po)) tc ,
 ST_CONTAINS(ST_GEOMETRYFROMTEXT(ST_ASTEXT(ST_Polygon("POLYGON ((116 30,117 30,117 32,116 32,116 30))"))), ST_GEOMETRYFROMTEXT(po)) tc2 
 from tmp ;

**Comment by yagagagaga** [comment]
中文问题，建议去 ask.selectdb.com 提问
